cloudinit/tests/helpers.py

```diff
@@ -509,6 +509,10 @@ def skipUnlessJsonSchema():
         _missing_jsonschema_dep, "No python-jsonschema dependency present.")
 
 
+def skipUnlessFreeBSD():
```

Comment:
The other `skipUnless` helpers are there for optional dependencies, not for differing platforms. You'll note that we don't have `skipUnlessUbuntu` or `skipUnlessFedora`, despite the fact that cloud-init exhibits different behaviour on those distributions. IMO, all of our unit tests should run on *BSD and any Linux distribution.

I'd happily accept this back alongside a compelling use, but I don't think keeping it around "just in case" is justifiable given the above.